ra/mock_test.go

```diff
@@ -2,210 +2,32 @@ package ra
 
 import (
 	"context"
+	"time"
 
-	"google.golang.org/grpc"
-
-	core "github.com/letsencrypt/boulder/core/proto"
-	corepb "github.com/letsencrypt/boulder/core/proto"
+	"github.com/letsencrypt/boulder/mocks"
 	sapb "github.com/letsencrypt/boulder/sa/proto"
 )
 
-var (
-	_ sapb.StorageAuthorityClient = &mockInvalidAuthorizationsAuthority{}
-)
-
 type mockInvalidAuthorizationsAuthority struct {
+	mocks.StorageAuthority
```

Comment:
Thanks for doing this cleanup! I think of it every time I add a new SA func and have to duplicate the mock here but hadn't got around to making a PR for it (shame on me).